spec/menu_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'menu'
+
+describe Menu do 
+    # User Story 1: As a customer
+    # So that I can check if I want to order something
+    # I would like to see a list of dishes with prices
```

Comment:
I like this use of comments. I feel it serves as a good reminder of what exactly you want the test to do.